test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1344,23 +1353,26 @@ class select_transport_rma_bw : public test_ucp_wireup {
 
 UCS_TEST_P(select_transport_rma_bw, select_rc, "NUM_EPS=56", "NUM_PPN=28")
 {
-    UCS_TEST_SKIP_R("FIXME: Disabled due to unresolved failure");
+    size_t ib_count = count_resource(&sender(), "rc_mlx5");
 
-    /* coverity[unreachable] */
-    if (!has_resource(&sender(), "rc_mlx5")) {
-        UCS_TEST_SKIP_R("no rc resources");
+    if (ib_count == 0) {
+        UCS_TEST_SKIP_R("no IB resources");
     }
 
     sender().connect(&receiver(), get_ep_params());
-
     const ucp_ep_config_t *config = ucp_ep_config(sender().ep());
-    for (int i = 0; (i < config->key.num_lanes) &&
-                    (config->key.rma_bw_lanes[i] != UCP_NULL_LANE);
-         ++i) {
-        /* Verify RC is selected for rma_bw lanes */
+
+    for (int i = 0; i < config->key.num_lanes; ++i) {
+        /* Only the first ib_count lanes are interesting, because
```

Comment:
`Only the first ib_count lanes are relevant` /
`Count only the first ib_count lanes`